### ISSUE TITLE
Irregular rotation (wobble) on hunter pet bar active abilities highlight

### ISSUE THREAD

**Issue by torhaala** [GoldpawsStuff/AzeriteUI_Classic#40]
## The problem  
On the hunter pet bar, the rotating ring signifying a skill is toggled On wobbles/shakes (not a smooth rotation).

Running a 1440p monitor, where WoW standard ui scale is set to 0.533333. 

### Make sure you can check all these boxes  
- [x] I have the latest addon version available from CurseForge/Twitch.
- [x] I restarted the entire game after updating the addon, not just relogged or reloaded. 

### Does this happen with NO other addons enabled?  

- [x] Yes
- [ ] No

### What classes does it happen on?  

- [ ] Druid 
- [s] Hunter 
- [ ] Mage 
- [ ] Paladin 
- [ ] Priest 
- [ ] Rogue 
- [ ] Shaman 
- [ ] Warlock 
- [ ] Warrior 

### Any logs, error output, etc?  
NA - But can make a gif to show if required.

**Comment by goldpaw** [comment]
It's a texture rotated in-game, and it's looking as I intended it. At least on my 1080p monitor. Can't say if it looks different on 1440p, so for that part a gif would be helpful. Not ruling out that something is wonky that I'm unaware of. 

What makes these things normally look smooth, is the speed and size. It would need to be larger, and move faster to have real smoothness to it, and that's not what I want. 

It's not a bug, though. So I'm changing the tag to "enhancement", if anything.

**Comment by torhaala** [comment]
The way i think its intended to look is great ;) It is just the slight shaking i assume is not as intended. I've attached a gif here of the action bar itself and how the skills look activated (shaking) and not activated.

![Petbar Issue](https://user-images.githubusercontent.com/13568322/84908822-40be3d80-b0b5-11ea-8ae6-d25f7a902d1d.gif)

**Comment by goldpaw** [comment]
Yeah that's not how it looks for me. It doesn't shake like that AT ALL! Definitely weird, and must be related to something with the scaling or positioning obviously.

Will look into it!

**Comment by goldpaw** [comment]
Looks like it's the blizzard animation system I'm using that just don't handle rotations very well. Will change it to be more of an oldschool animation instead, with the rotation done in advance in the graphics file. Will include the fix in the upcoming update!
